.github/workflows/deps_update.yml

```diff
@@ -0,0 +1,53 @@
+name: udd-update-dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "*/30 * * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: 1.15.3
+      - name: Update dependencies
+        run: |
+          deno run -A https://deno.land/x/udd/main.ts ./**/*.ts
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        id: pr
+        with:
+          commit-message: "chore: update deno dependencies"
+          title: "chore: update deno dependencies"
+          body: >
+            Dependencies updated by [udd](https://github.com/hayd/deno-udd).
+          branch: deno-dependency-updates
+          author: GitHub <noreply@github.com>
+          delete-branch: true
+      - name: Retrieve commit sha
+        id: commit
+        run: |
+          echo "::set-output name=sha::$(git rev-parse HEAD)"
+      - name: Set commit status with pending
+        uses: Sibz/github-status-action@v1
+        with: 
+          authToken: ${{ secrets.GITHUB_TOKEN }}
+          context: 'Basic tests'
+          state: 'pending'
+          sha: ${{ steps.commit.outputs.sha }}
+      - name: Basic tests
+        id: test
+        continue-on-error: true
+        run: |
+          deno test test/session.ts
```

Comment:
If you give your test files the `.test.ts` extension, they will be picked up automatically by running `deno test`.